src/bindings/python/src/pyopenvino/core/compiled_model.cpp

```diff
@@ -164,10 +161,10 @@ void regclass_CompiledModel(py::module m) {
 
     cls.def(
         "get_property",
-        [](ov::CompiledModel& self, const std::string& name) -> py::object {
-            return Common::utils::from_ov_any(self.get_property(name));
+        [](ov::CompiledModel& self, const py::object& property) -> py::object {
+            return Common::utils::property_as_py_object<ov::CompiledModel>(self, property);
         },
-        py::arg("name"),
+        py::arg("property"),
```

Comment:
won't it break compatibility?

Reply:
That is a very good question -- should we keep "name" just for sake of compatibility in kwagrs? New argument name is matching current `Union[str, Property]` possibility. Awaiting your feedback!

CC @AlexeyLebedev1

Reply:
@akuporos we decided to change it if possible and see if it breaks any of CIs. `property/properties` are more meaningful than simple `name`.

Reply:
I agree, but from my perspective it's a little bit late change it as our users may use named argument `name` in his/her code.